module/zfs/qat_compress.c

```diff
@@ -71,8 +71,13 @@ static CpaDcSessionHandle session_handles[MAX_INSTANCES];
 static CpaBufferList **buffer_array[MAX_INSTANCES];
 static Cpa16U num_inst = 0;
 static Cpa16U inst = 0;
-static int qat_init_done = 0;
-int zfs_qat_disable = B_FALSE;
+static boolean_t qat_init_done = B_FALSE;
+int zfs_qat_disable = 0;
```

Comment:
I tried, but seems module_param can't take a Boolean variable? Can you confirm it?